src/python/WMCore/ReqMgr/Service/Auxiliary.py

```diff
@@ -268,3 +269,12 @@ class PermissionsConig(AuxBaseAPI):
 
     def setName(self):
```

Comment:
since you're changing these files, can you fix a typo in here `PermissionsConig`?

Reply:
Thanks Alan, I will update that.